data/json/npcs/exodii/exodii_merchant_talk.json

```
@@ -623,7 +623,7 @@
     "type": "talk_topic",
     "dynamic_line": {
       "//~": "This one can't be saved too much brain rotted but i can salvage bits from them or repurpose them for walker, or just cut out rot and neural interface so you can bury them proper",
-      "str": "This 'unwon't be twitching', too much grey lost to dead.  But ye can salvage bits from 'em or born 'em for a walker, or just cut out the rot 'n grey link so ya can miss 'em proper."
+      "str": "This 'unwon't be twitching', too much gray lost to dead.  But ye can salvage bits from 'em or born 'em for a walker, or just cut out the rot 'n gray link so ya can miss 'em proper."
```

```suggestion
      "str": "This 'unwon't be twitching', too much grey lost to dead.  But ye can salvage bits from 'em or born 'em for a walker, or just cut out the rot 'n grey link so ya can miss 'em proper."
```
This character is canonically British adjacent.

---

An American transcribing what he says would still write `gray`.

---

It's good it's dialogue and not transcribed then.

---

I mean, I can see the argument: technically it's "transcribed" by the user interface. I guess it depends on whether the text we read is what the NPC says ("grey") or what our character hears ("gray").